mgo.go

```diff
@@ -295,7 +295,7 @@ func (inst *MgoInstance) run() error {
 }
 
 func getMongod() (string, error) {
-	paths := []string{"mongod", "/usr/lib/juju/bin/mongod"}
+	paths := []string{"mongod", "/usr/lib/juju/bin/mongod", "/usr/local/bin/mongod", "/usr/bin/mongod"}
```

Comment:
maybe add a comment explaining this is needed in tests where PATH is being completely removed